File: Baekjoon/2007_1924.py
month=[0,31,28,31,30,31,30,31,31,30,31,30,31]
week=['SUN','MON','TUE','WED','THU', 'FRI', 'SAT']

x,y=map(int,input().split())

day=0
for i in range(x):
    day+=month[i]
day+=y
print(week[day%7])

File: Baekjoon/2007_1924.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    int month[13]={0,31,28,31,30,31,30,31,31,30,31,30,31};
    string week[7]={"SUN","MON","TUE","WED","THU", "FRI", "SAT"};


    int x,y;
    cin >> x >> y;
    int day=0;

    for(int i=0;i<x;i++){
        day+=month[i];
    }
    day+=y;
    cout<<week[day%7];
}
